FastTimeSeries/Project.toml

```diff
@@ -4,18 +4,24 @@ authors = ["FluxML Community"]
 version = "0.1.0"
 
 [deps]
+DataDeps = "124859b0-ceae-595e-8997-d05f6a7a8dfe"
 FastAI = "5d0beca9-ade8-49ae-ad0b-a3cf890e669f"
 FilePathsBase = "48062228-2e41-5def-b9a4-89aafe57970f"
+Flux = "587475ba-b771-5e3f-ad9e-33799f191a9c"
 InlineTest = "bd334432-b1e7-49c7-a2dc-dd9149e4ebd6"
 MLUtils = "f1d291b0-491e-4a28-83b9-f70985020b54"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 UnicodePlots = "b8865327-cd53-5732-bb35-84acbb429228"
+Zygote = "e88e6eb3-aa80-5325-afca-941959d7151f"
```

Comment:
```suggestion
```
I don't see Zygote used anywhere, so one less dependency doesn't hurt.

Reply:
I tried doing that, but it throws an error 
```julia
ERROR: LoadError: ArgumentError: Package FastTimeSeries does not have Zygote in its dependencies:`
```

Reply:
This really could use a full stack trace, because AFAICT you're not using Zygote anywhere? Did you `] resolve` after removing it from the project?

Reply:
seems to work now, might have missed the `resolve`. thanks :)